tests/results/DiffTest.py

```diff
@@ -279,3 +279,17 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_diff_creation(self):
+        uut = Diff(create='create_test.py')
+        self.assertEqual(uut.create, 'create_test.py')
+        self.assertEqual(uut._file, [])
+        uut.add_lines(0, ['line2', 'line3'])
+        self.assertEqual(uut.modified, ['line2', 'line3'])
+
+        # Testing creation of a diff of not a new file
+        with self.assertRaises(ValueError) as err:
+            Diff()
+        self.assertEqual('file_list cannot be None.', err.exception.args[0])
```

Comment:
1. maybe you want to use `self.assertRaisesRegexp`
2. `err.exception.args[0]` --> `str(err.exception)`